packages/oui-back-button/src/back-button.component.js

```diff
@@ -7,13 +7,13 @@ export default {
     bindings: {
         id: "@?",
         name: "@?",
+        heading: "@?", // Deprecated: Replaced by transclude value
```

Comment:
:+1:

Reply:
Don't you keep the deprecated parameters in the documentation for a while ?

Reply:
Why not, we could do it for notifying that it will become deprecated